src/molecule_plugins/podman/playbooks/create.yml

```diff
@@ -39,6 +39,7 @@
         - item.registry is defined
         - item.registry.credentials is defined
         - item.registry.credentials.username is defined
+      changed_when: false
```

Comment:
are you sure it's the proper way to do that ? this command (and the next one) are clearly doing something on the system. Unless podman login or build are doing nothing ?

Reply:
Thanks for pointing it out.

IMO, if the login fails, the playbook would fail. 
Instead, if the login run successfully, there would be a bit noise if the status is changed. So the changed_when is set to false.

Does that make sense?